python/servo/package_commands.py

```diff
@@ -179,6 +188,50 @@ def package(self, build_type: BuildType, flavor=None, with_asan=False):
             except subprocess.CalledProcessError as e:
                 print("Packaging Android exited with return value %d" % e.returncode)
                 return e.returncode
+        elif self.is_openharmony():
+            # hvigor doesn't support an option to place output files in a specific directory
+            # so copy the source files into the target/openharmony directory first.
+            ohos_app_dir = path.join(self.get_top_dir(), "support", "openharmony")
+            ohos_target_dir = path.join(
+                self.get_top_dir(), "target", "openharmony", self.target.triple(), build_type.directory_name())
+            if path.exists(ohos_target_dir):
+                print("Cleaning up from previous packaging")
+                delete(ohos_target_dir)
+            shutil.copytree(ohos_app_dir, ohos_target_dir)
+
+            hvigor_command = ["--no-daemon", "assembleHap", "-p", "product=default"]
```

Comment:
I guess here we have the difficulty that from the target-triple alone, we can't deduce if the user wants to build for OpenHarmony or HarmonyOS.
I guess for now It's fine to just support the default OH build.

Reply:
For Android we have the `--flavor` argument to choose the variant. I was hoping to reuse that eventually, but I have not tested the HarmonyOS variant and it also requires different environment variables during build (DEVECO_SDK_HOME vs OHOS_BASE_SDK_HOME) so this currently defaults to OH.

Reply:
For HOS hvigor5 also seems to require the whole HOS commandline tools package, which I don't think we can currently download in CI. I'll raise that question internally.